### PR TITLE
feat(optimus): add namespace name to RegisterSecretRequest

### PR DESCRIPTION
**Pull request by arinda-arif**

### CODE REVIEW

File: odpf/optimus/runtime_service.proto [hunk 1]
```diff
@@ -554,6 +554,8 @@ message RegisterSecretRequest {
     string project_name = 1;
     string secret_name = 2;
     string value = 3; // base64 encoded secret value
+    string namespace_name = 4;
+    bool updateOnly = 5;
```
Comment (kushsharma) [inline odpf/optimus/runtime_service.proto:558]: this should not be camel case

**Review by kushsharma — CHANGES_REQUESTED** [review]

File: odpf/optimus/runtime_service.proto (hunk 1 above)
Comment (sbchaos) [reply odpf/optimus/runtime_service.proto:558]: Thank you for pointing out. I will make this change.

File: odpf/optimus/core/v1beta1/runtime.proto [hunk 2]
```diff
@@ -559,13 +568,26 @@ message RegisterSecretRequest {
...
+message UpdateSecretRequest {
+    string project_name = 1;
+    string secret_name = 2;
+    string value = 3; // base64 encoded secret value
+    string namespace_name = 4;
+}
+
+message UpdateSecretResponse {
+    bool success = 1;
+    string message = 2;
```
Comment (kushsharma) [inline odpf/optimus/core/v1beta1/runtime.proto:588]: How are we using these? I don't think we need a success flag here?

File: odpf/optimus/runtime_service.proto (hunk 1 above)
Comment (sbchaos) [reply odpf/optimus/runtime_service.proto]: Marking as resolved, changes already done.

File: odpf/optimus/core/v1beta1/runtime.proto (hunk 2 above)
Comment (sbchaos) [reply odpf/optimus/core/v1beta1/runtime.proto:588]: Sure, will remove `success` flag from the proto in next PR.

**Pull request merged**